src/utils/logMessageFilter.ts

```diff
@@ -0,0 +1,52 @@
+import { ActivityType, IDirectLineActivity, Role, TwoWaySerializableComplex } from '../types/DirectLineTypes';
+
+export interface LogDLActivity {
+    attachmentsCount?: number;
+    channelData?: TwoWaySerializableComplex;
+    channelId: string;
+    conversation: {
+        id: string;
+    };
+    from: {
+        role?: Role;
+    };
+    textLength?: number;
+    id?: string;
+    suggestedActionCount?: number;
+    timestamp: string;
+    type: ActivityType;
+    hasValue?: boolean;
+    messageid?: string;
+    previousClientActivityID?: string;
+}
+export function logMessagefilter(message: IDirectLineActivity): string {

```

Comment:
I fear this is going to overload the database...

Reply:
Should be OK. compared to the number of notification updating messages, this is only a small piece.